Vagrantfile

```diff
@@ -18,7 +18,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.network :private_network, ip: vconfig['vagrant_ip']
   config.ssh.insert_key = false
 
-  config.vm.box = "geerlingguy/ubuntu1404"
+  vconfig['vagrant_os'] ||= "ubuntu14"
```

Comment:
For this, I'd rather add a config option `vagrant_box`, and default it to `geerlingguy/ubuntu1404`. That way users could fork my own Packer configurations and create their own unique boxes—this would especially come in handy if they wanted to use Vagrant to provision on other cloud providers too...

So this line would just be `config.vm.box = vconfig['vagrant_box']`